bashrc

```diff
@@ -0,0 +1,8 @@
+# Note that this file is overridden by newer versions of `nextstrain shell` and
+# so it provides only a fallback prompt for users of older CLI versions or
+# direct image users (e.g. `docker run …`).
```

Comment:
This comment is a bit forward-looking to the WIP `nextstrain shell` improvements re: the new managed Conda runtime.